thread-manager/src/default_config.toml

```diff
@@ -0,0 +1,99 @@
+# pinned threads that do very important stuff
+[native_configs.solPohTickProd] # PoH thread
+max_threads = 1
+core_allocation.PinnedCores = { min = 0, max = 1 }
+
+[native_configs.solSigVerTpuVote]
+max_threads = 1
+
+[native_configs.solRcvrGossip] # Runs gossip receiver thread which seems to eat a lot of CPU all the time
+max_threads = 1
+core_allocation.PinnedCores = { min = 2, max = 3 }
+
+[native_configs.solSigVerTpu]
+max_threads = 1
+
+
+[native_configs.solClusterInfo]
+max_threads = 3
+
+
+[native_configs.solGossip]
+max_threads = 16
+
+[native_configs.solRepair]
+max_threads = 16
+
+[native_configs.FetchStage]
+max_threads = 16
+
+[native_configs.solShredFetch]
+max_threads = 16
+
+
+[rayon_configs.solReplayTx] #Replay transactions pool
+worker_threads = 12
+
+[rayon_configs.solReplayFork] # Replay forks pool
+worker_threads = 16
+
+[rayon_configs.solSvrfyShred] # runs turbine protocol signature verification for received shreds
+max_threads = 12
+
+[rayon_configs.solSigVerify] # Runs signature verification for cluster info
+max_threads = 8
+
+[rayon_configs.solRetransmit] # runs turbine protocol retransmit logic
+max_threads = 4
+
+[rayon_configs.solRunGossip]
+worker_threads = 4
+
+[rayon_configs.solGossipWork]
+worker_threads = 4
+
+[rayon_configs.solWinInsert]
+worker_threads = 2
+
+[rayon_configs.solGossipCons] # Gossip Consume
+worker_threads = 16
+
+[rayon_configs.solAccounts] # AccountsDb foreground threads
+worker_threads = 6
+
+stack_size_bytes = 8388608 # 8 megabytes
+
+[rayon_configs.solAccountsLo] # AccountsDb backkground
+worker_threads = 4
+stack_size_bytes = 8388608 # 8 megabytes
+
+[rayon_configs.solAcctHash] # AccountsDb hash pool
+worker_threads = 8
+
+[rayon_configs.solVoteSigVerTpu] # TPU vote sigverify stage
+worker_threads = 4
+
+[rayon_configs.solTrSigVerTpu] # TPU transaction sigverify stage
+worker_threads = 4
+
+# Networking runtimes (tokio)
+[tokio_configs.solQuicClientRt]
+worker_threads = 4
+
+[tokio_configs.solQuicTVo]
+worker_threads = 4
+
+[tokio_configs.solQuicTpu]
+worker_threads = 4
+
+[tokio_configs.solQuicTpuFwd]
+worker_threads = 4
+
+[tokio_configs.solRepairQuic] # this is not used on MNB
```

Comment:
these comments are getting orphaned for sure if/when we turn on QUIC for these protocols 😆

Reply:
Yes:) But I will re add stuff to the file as I run integration and make sure they are all correct.